src/libraries/Microsoft.PowerFx.Connectors/Public/CdpExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    public static class CdpExtensions
```

Comment:
>public [](http://example.com/codeflow?start=4&length=6)

nit: Can we keep it internal? or if we need to keep it public can we add comment on the below method.

Reply:
nope, I really want this API to be public and that's the reason for this extension as the CdpRecordType class is internal